### PR TITLE
clp and osi depend on pkg-config

### PR DESCRIPTION
**Pull request by vmiheer**

### CODE REVIEW

File: var/spack/repos/builtin/packages/clp/package.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ class Clp(AutotoolsPackage):
 
     depends_on('coinutils')
     depends_on('osi')
+    depends_on('pkg-config')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/clp/package.py]:
```suggestion
    depends_on("pkg-config", type="build")
```

File: var/spack/repos/builtin/packages/osi/package.py [hunk 2]
```diff
@@ -21,6 +21,7 @@ class Osi(AutotoolsPackage):
     url      = "https://github.com/coin-or/Osi/archive/releases/0.108.6.tar.gz"
 
     depends_on('coinutils')
+    depends_on('pkg-config')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/osi/package.py]:
```suggestion
    depends_on("pkg-config", type="build")
```

**Review by alalazo — CHANGES_REQUESTED** [review]
`pkg-config` is usually a build dependency

**Comment by alalazo** [comment]
Can you also resolve the conflicts?

**Review by alalazo — DISMISSED** [review]

File: var/spack/repos/builtin/packages/clp/package.py [hunk 3]
```diff
@@ -15,6 +15,7 @@ class Clp(AutotoolsPackage):
 
     depends_on("coinutils")
     depends_on("osi")
+    depends_on("pkg-config", type="build")
```
Comment (haampie) [inline var/spack/repos/builtin/packages/clp/package.py]:
```suggestion
    depends_on("pkgconfig", type="build")
```

File: var/spack/repos/builtin/packages/osi/package.py [hunk 4]
```diff
@@ -21,6 +21,7 @@ class Osi(AutotoolsPackage):
     url = "https://github.com/coin-or/Osi/archive/releases/0.108.6.tar.gz"
 
     depends_on("coinutils")
+    depends_on("pkg-config", type="build")
```
Comment (haampie) [inline var/spack/repos/builtin/packages/osi/package.py]:
```suggestion
    depends_on("pkgconfig", type="build")
```

**Review by haampie — DISMISSED** [review]
pkgconfig is the provider, pkg-config and pkgconf the implementations.

**Review by alalazo — APPROVED** [review]

**Comment by vmiheer** [comment]
Thank you, @alalazo!!!

**Pull request merged**